lib/i18n/hygiene/checks/html_entity.rb

```diff
@@ -8,22 +8,12 @@ module Hygiene
     module Checks
       class HtmlEntity < Base
         def run
-          puts "Checking for phrases that contain entities but probably shouldn't..."
-
           wrapper = I18n::Hygiene::Wrapper.new(locales: all_locales)
 
           keys_with_entities = I18n::Hygiene::KeysWithEntities.new(i18nwrapper: wrapper)
 
           keys_with_entities.each do |key|
```

Comment:
There's an opportunity here to use the one line, idiomatic form of a block that contains a single statement.